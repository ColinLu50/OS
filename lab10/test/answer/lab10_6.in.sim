7712
231
233
143
231
142
