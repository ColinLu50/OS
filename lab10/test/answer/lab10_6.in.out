FCFS
88 -> 68 -> 23 -> 46 -> 52 -> 48 -> 15 -> 74 -> 88 -> 138 -> 13 -> 132 -> 112 -> 117 -> 56 -> 4 -> 118 -> 142 -> 39 -> 116 -> 106 -> 13 -> 107 -> 134 -> 108 -> 112 -> 55 -> 116 -> 35 -> 21 -> 10 -> 83 -> 106 -> 30 -> 86 -> 76 -> 121 -> 87 -> 32 -> 79 -> 34 -> 77 -> 138 -> 128 -> 90 -> 109 -> 72 -> 14 -> 63 -> 13 -> 137 -> 86 -> 56 -> 24 -> 23 -> 82 -> 65 -> 70 -> 73 -> 108 -> 78 -> 30 -> 138 -> 96 -> 141 -> 31 -> 137 -> 35 -> 76 -> 71 -> 38 -> 38 -> 9 -> 26 -> 25 -> 59 -> 3 -> 68 -> 82 -> 86 -> 69 -> 58 -> 145 -> 30 -> 34 -> 43 -> 139 -> 121 -> 96 -> 58 -> 28 -> 23 -> 21 -> 6 -> 107 -> 83 -> 42 -> 8 -> 36 -> 32 -> 145 -> 29 -> 144 -> 130 -> 125 -> 90 -> 17 -> 73 -> 13 -> 26 -> 38 -> 93 -> 55 -> 90 -> 38 -> 104 -> 99 -> 92 -> 27 -> 142 -> 103 -> 126 -> 26 -> 88 -> 47 -> 112 -> 33 -> 127 -> 78 -> 123 -> 134 -> 78 -> 129 -> 13 -> 130 -> 22 -> 80 -> 1 -> 71 -> 9 -> 53 -> 61 -> 47 -> 128 -> 120 -> 95 -> 67 -> 76 -> 113 -> 104 -> 112 -> 36 -> 16 -> 134 -> 44 -> 75 -> 18 -> 128 -> 45 -> 34 -> 126 -> 49 -> 60 -> 87 -> 36
7712
SSTF
88 -> 88 -> 88 -> 87 -> 87 -> 86 -> 86 -> 86 -> 83 -> 83 -> 82 -> 82 -> 80 -> 79 -> 78 -> 78 -> 78 -> 77 -> 76 -> 76 -> 76 -> 75 -> 74 -> 73 -> 73 -> 72 -> 71 -> 71 -> 70 -> 69 -> 68 -> 68 -> 67 -> 65 -> 63 -> 61 -> 60 -> 59 -> 58 -> 58 -> 56 -> 56 -> 55 -> 55 -> 53 -> 52 -> 49 -> 48 -> 47 -> 47 -> 46 -> 45 -> 44 -> 43 -> 42 -> 39 -> 38 -> 38 -> 38 -> 38 -> 36 -> 36 -> 36 -> 35 -> 35 -> 34 -> 34 -> 34 -> 33 -> 32 -> 32 -> 31 -> 30 -> 30 -> 30 -> 29 -> 28 -> 27 -> 26 -> 26 -> 26 -> 25 -> 24 -> 23 -> 23 -> 23 -> 22 -> 21 -> 21 -> 18 -> 17 -> 16 -> 15 -> 14 -> 13 -> 13 -> 13 -> 13 -> 13 -> 10 -> 9 -> 9 -> 8 -> 6 -> 4 -> 3 -> 1 -> 90 -> 90 -> 90 -> 92 -> 93 -> 95 -> 96 -> 96 -> 99 -> 103 -> 104 -> 104 -> 106 -> 106 -> 107 -> 107 -> 108 -> 108 -> 109 -> 112 -> 112 -> 112 -> 112 -> 113 -> 116 -> 116 -> 117 -> 118 -> 120 -> 121 -> 121 -> 123 -> 125 -> 126 -> 126 -> 127 -> 128 -> 128 -> 128 -> 129 -> 130 -> 130 -> 132 -> 134 -> 134 -> 134 -> 137 -> 137 -> 138 -> 138 -> 138 -> 139 -> 141 -> 142 -> 142 -> 144 -> 145 -> 145
231
SCAN
88 -> 87 -> 86 -> 83 -> 82 -> 80 -> 79 -> 78 -> 77 -> 76 -> 75 -> 74 -> 73 -> 72 -> 71 -> 70 -> 69 -> 68 -> 67 -> 65 -> 63 -> 61 -> 60 -> 59 -> 58 -> 56 -> 55 -> 53 -> 52 -> 49 -> 48 -> 47 -> 46 -> 45 -> 44 -> 43 -> 42 -> 39 -> 38 -> 36 -> 35 -> 34 -> 33 -> 32 -> 31 -> 30 -> 29 -> 28 -> 27 -> 26 -> 25 -> 24 -> 23 -> 22 -> 21 -> 18 -> 17 -> 16 -> 15 -> 14 -> 13 -> 10 -> 9 -> 8 -> 6 -> 4 -> 3 -> 1 -> 0 -> 90 -> 92 -> 93 -> 95 -> 96 -> 99 -> 103 -> 104 -> 106 -> 107 -> 108 -> 109 -> 112 -> 113 -> 116 -> 117 -> 118 -> 120 -> 121 -> 123 -> 125 -> 126 -> 127 -> 128 -> 129 -> 130 -> 132 -> 134 -> 137 -> 138 -> 139 -> 141 -> 142 -> 144 -> 145
233
C-SCAN
88 -> 87 -> 86 -> 83 -> 82 -> 80 -> 79 -> 78 -> 77 -> 76 -> 75 -> 74 -> 73 -> 72 -> 71 -> 70 -> 69 -> 68 -> 67 -> 65 -> 63 -> 61 -> 60 -> 59 -> 58 -> 56 -> 55 -> 53 -> 52 -> 49 -> 48 -> 47 -> 46 -> 45 -> 44 -> 43 -> 42 -> 39 -> 38 -> 36 -> 35 -> 34 -> 33 -> 32 -> 31 -> 30 -> 29 -> 28 -> 27 -> 26 -> 25 -> 24 -> 23 -> 22 -> 21 -> 18 -> 17 -> 16 -> 15 -> 14 -> 13 -> 10 -> 9 -> 8 -> 6 -> 4 -> 3 -> 1 -> 0 -> 145 -> 145 -> 144 -> 142 -> 141 -> 139 -> 138 -> 137 -> 134 -> 132 -> 130 -> 129 -> 128 -> 127 -> 126 -> 125 -> 123 -> 121 -> 120 -> 118 -> 117 -> 116 -> 113 -> 112 -> 109 -> 108 -> 107 -> 106 -> 104 -> 103 -> 99 -> 96 -> 95 -> 93 -> 92 -> 90
143
LOOK
88 -> 87 -> 86 -> 83 -> 82 -> 80 -> 79 -> 78 -> 77 -> 76 -> 75 -> 74 -> 73 -> 72 -> 71 -> 70 -> 69 -> 68 -> 67 -> 65 -> 63 -> 61 -> 60 -> 59 -> 58 -> 56 -> 55 -> 53 -> 52 -> 49 -> 48 -> 47 -> 46 -> 45 -> 44 -> 43 -> 42 -> 39 -> 38 -> 36 -> 35 -> 34 -> 33 -> 32 -> 31 -> 30 -> 29 -> 28 -> 27 -> 26 -> 25 -> 24 -> 23 -> 22 -> 21 -> 18 -> 17 -> 16 -> 15 -> 14 -> 13 -> 10 -> 9 -> 8 -> 6 -> 4 -> 3 -> 1 -> 90 -> 92 -> 93 -> 95 -> 96 -> 99 -> 103 -> 104 -> 106 -> 107 -> 108 -> 109 -> 112 -> 113 -> 116 -> 117 -> 118 -> 120 -> 121 -> 123 -> 125 -> 126 -> 127 -> 128 -> 129 -> 130 -> 132 -> 134 -> 137 -> 138 -> 139 -> 141 -> 142 -> 144 -> 145
231
C-LOOK
88 -> 87 -> 86 -> 83 -> 82 -> 80 -> 79 -> 78 -> 77 -> 76 -> 75 -> 74 -> 73 -> 72 -> 71 -> 70 -> 69 -> 68 -> 67 -> 65 -> 63 -> 61 -> 60 -> 59 -> 58 -> 56 -> 55 -> 53 -> 52 -> 49 -> 48 -> 47 -> 46 -> 45 -> 44 -> 43 -> 42 -> 39 -> 38 -> 36 -> 35 -> 34 -> 33 -> 32 -> 31 -> 30 -> 29 -> 28 -> 27 -> 26 -> 25 -> 24 -> 23 -> 22 -> 21 -> 18 -> 17 -> 16 -> 15 -> 14 -> 13 -> 10 -> 9 -> 8 -> 6 -> 4 -> 3 -> 1 -> 145 -> 144 -> 142 -> 141 -> 139 -> 138 -> 137 -> 134 -> 132 -> 130 -> 129 -> 128 -> 127 -> 126 -> 125 -> 123 -> 121 -> 120 -> 118 -> 117 -> 116 -> 113 -> 112 -> 109 -> 108 -> 107 -> 106 -> 104 -> 103 -> 99 -> 96 -> 95 -> 93 -> 92 -> 90
142
